### PR TITLE
Migrate system user

### PR DESCRIPTION
**Pull request by phlax**
Currently the system user is created by initdb - but other code relies on system user existing whether this has been run or not.

This PR creates the system user in a migration where required.

### CODE REVIEW

File: pootle/core/initdb.py [hunk 1]
```diff
@@ -110,15 +110,6 @@ def create_essential_users(self):
         }
         self._create_pootle_user(**criteria)
 
-        # The system user represents a system, and is used to
-        # associate updates done by bulk commands as update_stores.
-        criteria = {
-            'username': u"system",
-            'full_name': u"Pootle",
-            'is_active': True,
```
Comment (ta2-1) [inline pootle/core/initdb.py]: @phlax `is_active` is not set in the new code?

Comment (phlax) [inline pootle/core/initdb.py]: thanks, ill check...

File: pootle/apps/accounts/migrations/0008_add_system_user.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-02 19:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def add_system_user(apps, schema_editor):
+    User = apps.get_model("accounts.User")
+    sysuser = users.filter(username="system").first()
```
Comment (unho) [inline pootle/apps/accounts/migrations/0008_add_system_user.py]: Use `exists()`

**Comment by unho** [comment]
lgtm

**Pull request merged**